Formula/r/rofi.rb

```diff
@@ -0,0 +1,42 @@
+class Rofi < Formula
+  desc "Window switcher, application launcher and dmenu replacement"
+  homepage "https://davatorium.github.io/rofi/"
+  url "https://github.com/davatorium/rofi/releases/download/1.7.8/rofi-1.7.8.tar.gz"
+  sha256 "469fba08ad99f286a4a8c65f857c9f66c07e7b9a3496ac1fe3dcd856f3c687d3"
+  license "MIT"
+  head "https://github.com/davatorium/rofi.git", branch: "next"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "check" => :build
+  depends_on "pkgconf" => :build
+  depends_on "cairo"
+  depends_on "gdk-pixbuf"
+  depends_on "gettext"
+  depends_on "glib"
+  depends_on "harfbuzz"
+  depends_on "libxcb"
+  depends_on "libxkbcommon"
+  depends_on "pango"
+  depends_on "startup-notification"
+  depends_on "xcb-util"
+  depends_on "xcb-util-cursor"
+  depends_on "xcb-util-keysyms"
```

Comment:
this should be macos only dependency

Reply:
All of these 4?

Reply:
The linkage test (the screenshot Rui commented above) didn't show any linkage on Linux.

Reply:
Ah yes, thanks. Just `xcb-util-keysyms` then.